server/sonar-web/src/main/js/apps/issues/components/App.js

```diff
@@ -182,26 +195,64 @@ export default class App extends React.PureComponent {
       this.closeIssue();
       return false;
     });
+    window.addEventListener('keydown', this.handleKeyDown);
+    window.addEventListener('keyup', this.handleKeyUp);
   }
 
   detachShortcuts() {
     key.deleteScope('issues');
+    window.removeEventListener('keydown', this.handleKeyDown);
+    window.removeEventListener('keyup', this.handleKeyUp);
   }
 
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (key.getScope() !== 'issues') {
+      return;
+    }
+    if (event.keyCode === 18) {
+      // alt
+      event.preventDefault();
```

Comment:
I'm not sure we can do anything about it, but when clicking `alt` only, on ubuntu, I have the command shortcut that is triggered and that makes me loose focus on the browser.